lib/req_chronicle/options.ex

```diff
@@ -5,7 +5,13 @@ defmodule ReqChronicle.Options do
 
   body_handler_definition = [
     type: :mfa,
-    default: {Kernel, :inspect, []},
+    default: {Kernel, :inspect, [
+      [
+        limit: :infinity,
+        pretty: true,
```

Comment:
I'm in two minds of setting this by default, it will result in more text control characters in the output but would make it easier when I'm debugging what's going on. It does seem like the kind of thing that could / should be set on an individual implementers basis though.